tests/foreman/ui/test_contentview.py

```diff
@@ -1289,8 +1382,9 @@ def test_positive_publish_with_custom_content(self):
             self.assertIsNotNone(self.content_views.wait_until_element(
                 common_locators['alert.success_sub_form']))
 
-    @stubbed()
+    @run_in_one_thread
     @run_only_on('sat')
+    @skip_if_not_set('fake_manifest')
     @tier2
     def test_positive_publish_composite_with_custom_content(self):
```

Comment:
@jyejare, in some part you are right, but in the first test we promote the composite content view in the second we promote one of the base content view.